test/test-50-reproducible/main.js

```diff
@@ -20,6 +20,7 @@ const output2 = './test-output-2.exe';
 
 utils.pkg.sync([
   '--public',
+  '--no-signature', // the signature will make the build not reproducible
```

Comment:
how did this work before? is this just applicable to running test-50-reproducible on macos now?